src/implementations/twig/components/site-header/site-header.html.twig

```diff
@@ -317,6 +324,33 @@
             } only %}
           {% endif %}
 
+          {# Basket #}
+          {% if _basket is not empty %}
+            <div class="ecl-site-header__basket">
+              <a
```

Comment:
i guess there is a reason for this, why not including link here instead of defining the whole markup in this template?

Reply:
the main reason is that having the class ecl-link would alter the display (color, underline, ...), and so would require some additional overrides